src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,13 +1,55 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private String maker;
+    private String model;
+    private int year;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, String maker, String model, int year) {
+        setColor(color);
+        setName(name);
+        this.maker = maker;
+        this.model = model;
+        this.year = year;
+    }
+
+    public String getMaker() {
+        return maker;
+    }
+
+    public void setMaker(String maker) {
+        this.maker = maker;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public int getYear() {
+        return year;
+    }
+
+    public void setYear(int year) {
+        this.year = year;
+    }
+
+    @Override
+    public void setName(String name) {
+        super.setName(name);
+    }
+
+    @Override
+    public void setColor(String color) {
+        super.setColor(color);
+    }
```

Comment:
You can invoke these methods from the parent's class without this overridden method.